pkg/apis/kops/validation/instancegroup.go

```diff
@@ -126,6 +126,11 @@ func ValidateInstanceGroup(g *kops.InstanceGroup, cloud fi.Cloud) field.ErrorLis
 		allErrs = append(allErrs, validateCloudLabels(g, field.NewPath("spec", "cloudLabels"))...)
 	}
 
+	// InstanceMetadata is currently supported only for AWS cloud to require EC2 InstanceMetadata v2
+	if g.Spec.InstanceMetadata != nil {
+		allErrs = append(allErrs, validateInstanceMetadata(g, cloud, field.NewPath("spec", "instanceMetadata"))...)
+	}
+
```

Comment:
At the moment we don't validate for AWS, just don't apply it. Please remove these checks.

Reply:
yes. will remove the checks.